sample/src/androidTest/java/com/schibsted/spain/barista/sample/RadioButtonsTest.java

```diff
@@ -38,4 +40,16 @@ public void checkRadioButtonsByText_secondItem() {
     clickRadioButtonItem(R.id.radiogroup, "Bye bye");
     assertDisplayed("" + R.id.second_item);
   }
+
+  @Test
+  public void checkRadioButtonsByPosition_firstItem() {
+    clickRadioButtonPosition(R.id.radiogroup, 0);
+    assertDisplayed("" + R.id.first_item);
```

Comment:
It's the same way we're checking that the ID of that item has been clicked. If you have a better idea... we could implement it in another PR, and change other methods as well :·D